tests/click_elements.py

```diff
@@ -1,17 +1,21 @@
 # Copyright 2012 splinter authors. All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
+from .fake_webapp import EXAMPLE_APP
 
 
 class ClickElementsTest:
     def test_click_links(self):
         self.browser.links.find_by_text("FOO").click()
         self.assertIn("BAR!", self.browser.html)
+        self.browser.visit(EXAMPLE_APP)
```

Comment:
why did we need to make this change?

Reply:
The visit in `setUp` was unstable for some reason and page just stuck. Now seems ok. Removed all `visit` in tests